rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/NSCreationServlet.java

```diff
@@ -114,11 +116,9 @@ private void createNs(HttpServletRequest request, HttpServletResponse response)
                     } else if (formFieldName.equals("nodesRecoverable")) {
                         nodesRecoverable = formFieldValue;
                     } else if (formFieldName.equals("deploy")) {
-                        if (formFieldValue.equals(Boolean.TRUE.toString())) {
-                            deployNodeSource = true;
-                        }
+                        deployNodeSource = formFieldValue.equals(Boolean.TRUE.toString());
```

Comment:
May be use equalsIgnoreCase() to compare "formFieldName" and "deploy".

Reply:
Yes, same reply as per Amine's comment. Thank you.